tests/migration/online_migration/yast2_migration.pm

```diff
@@ -171,13 +172,13 @@ sub run {
     }
 
     # wait for migration target after needed updates installed
-    assert_screen ['yast2-migration-target', 'yast2-migration-inconsistency'], 300;
+    assert_screen ['yast2-migration-target', 'yast2-migration-inconsistency', 'yast2-migration-leap_to_sle'], 300;
     if (match_has_tag 'yast2-migration-inconsistency') {
         if (get_var('MIGRATION_INCONSISTENCY_DEACTIVATE')) {
             send_key 'alt-d';    # deactivate
         }
         elsif (get_var('MIGRATION_INCONSISTENCY_INSTALL')) {
-            wait_screen_change { send_key 'alt-t' };    # install product
+            wait_screen_change { send_key 'alt-t' };
```

Comment:
I think reserve the comments will be helpful for code understanding.

Reply:
changed, thanks